Libraries/Text/Text.js

```diff
@@ -317,6 +326,12 @@ const Text = React.createClass({
         isHighlighted: this.state.isHighlighted,
       };
     }
+    if (typeof newProps.selectionColor !== 'undefined') {
```

Comment:
@janicduplessis `Text` doesn't use the `requireNativeComponent` which calls `processColor`, so this seems to be needed.